### DISCUSSION TITLE
Add a custom menu button in existing controls group of a Default layout - React

### DISCUSSION THREAD

**Discussion by simo54s** [vidstack/player#1313] [Q&A] [answered]
As referenced in [docs](https://www.vidstack.io/docs/player/components/menus/menu?styling=default-theme) I want to add an additional button for the default menu

This is my attempt so far => https://stackblitz.com/edit/vidstack-examples-is7gzr?file=src%2Fplayer.tsx&showSidebar=1&title=Vidstack%20Player%20-%20React%20(Default%20Layout)

I managed to add an icon before the settings button using slot property <code>beforeSettingsMenu</code> but I don't know how to config any trigger to open menu/submenu, plus it does not have the same look and feel of the other buttons. 

I can't find any example (only in a custom layout using [radix+tailwind](https://github.com/vidstack/examples/tree/main/player/react/radix%2Btailwind)) and the only reference I could find was in [this thread](https://github.com/vidstack/player/discussions/1136).

Can you please advise or point out the docs where they mention this?

**Comment by mihar-22** [comment, marked as answer]
Creating a menu with the default theme is documented [here](https://www.vidstack.io/docs/player/components/menus/menu) and [here](https://github.com/vidstack/examples/blob/main/player/react/default-theme/src/layouts/shared/menus.tsx#L22-L40).

**Reply by simo54s** [reply]
@mihar-22 thnx for the reply, I ended up following [this](https://github.com/vidstack/examples/blob/main/player/react/default-theme/src/layouts/shared/menus.tsx#L22-L40) by recreating the files and adding the custom icon within <code>Controls.Group</code>

**Reply by trry-hub** [reply]
Can it be implemented in Vue? I want to customize the media time slider component
